.github/workflows/release-canary.yaml

```diff
@@ -50,10 +50,11 @@ jobs:
       - name: Authenticate to npm and publish
         run: |
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > .npmrc
-          pnpm publish packages/uploadthing --access public --tag canary --no-git-checks
-          pnpm publish packages/mime-types --access public --tag canary --no-git-checks
-          pnpm publish packages/shared --access public --tag canary --no-git-checks
-          pnpm publish packages/react --access public --tag canary --no-git-checks
+          for dir in packages/*/; do
+            if [ "$dir" != "packages/config/" ]; then
+              pnpm publish "$dir" --access public --tag canary --no-git-checks
```

Comment:
Will $dir include packages/ here?

Reply:
If you look at the previous runs, it seems to be working, so yes? 🤷‍♂️

Reply:
yup!